docs/docusaurus/docs/core/trigger_actions_based_on_results/create_a_checkpoint_with_actions.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Create a Checkpoint with Actions
+---
+import TabItem from '@theme/TabItem';
+import Tabs from '@theme/Tabs';
+
+import PrereqPythonInstalled from '../_core_components/prerequisites/_python_installation.md';
+import PrereqGxInstalled from '../_core_components/prerequisites/_gx_installation.md';
+import PrereqPreconfiguredDataContext from '../_core_components/prerequisites/_preconfigured_data_context.md';
+import PrereqValidationDefinition from '../_core_components/prerequisites/_validation_definition.md';
+
+A Checkpoint executes one or more Validation Definitions and then performs a set of Actions based on the Validation Results each Validation Definition returns.
+
+<h2>Prerequisites</h2>
+
+- <PrereqPythonInstalled/>
+- <PrereqGxInstalled/>
+- <PrereqPreconfiguredDataContext/>
+- <PrereqValidationDefinition/>
+
+<Tabs>
+
+<TabItem value="procedure" label="Procedure">
+
+1. Retrieve the Validation Definitions the Checkpoint will run.
+
+   In the following example your Data Context has already been stored in the variable `context` and an existing Validation Definition is retrieved from it:
+
+   ```python title="Python"
+   validation_definitions=[
+      # A pre-existing Validation Definition
+      context.validation_definitions.get("my_validation_definition")
+   ]
+   ```
+
+2. Determine the Actions that the Checkpoint will automate.
+
+   After a Checkpoint receives Validation Results from a Validation Definition, it executes a list of Actions. The returned Validation Results determine what task is performed for each Action. Actions can include updating Data Docs with the new Validation Results or sending alerts when validations fail.  The Actions list is executed once for each Validation Definition in a Checkpoint.
```

Comment:
I find `receives` a bit confusing here. Maybe make this explicit that this is from running a validation? That is:

After a Checkpoint receives Validation Results from **running** a Validation Definition,